example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -366,7 +364,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
If this is targeting SDK 12, the target should be 11.

Reply:
we aren't targeting SDK 12 yet